provider/core/src/varule_traits.rs

```diff
@@ -121,3 +121,14 @@ impl<T, const N: usize> MaybeEncodeAsVarULE for [T; N] {
         None
     }
 }
+
+#[cfg(feature = "export")]
+impl MaybeEncodeAsVarULE for u16 {
+    fn maybe_encode_as_varule(&self) -> Option<&Self::EncodedStruct> {
+        None
+    }
+}
+
+impl MaybeAsVarULE for u16 {
```

Comment:
Nit: Keep the impls in the same order as above. `MaybeAsVarULE` should go above `MaybeEncodeAsVarULE`.

Reply:
done